app/TestRocketApp.vue

```diff
@@ -1,40 +1,117 @@
 <template>
     <div class="p-4">
-        <h1 class="text-xl font-bold">Users lijst</h1>
+        <h1 class="text-xl font-bold">Users list</h1>
         <ul>
             <li :key="user.email" v-for="user of users">
                 {{ user.full_name }}
             </li>
         </ul>
-        <button class="button">Nutteloze button</button>
+
+        <form @submit.prevent="submit()" class="border-t py-4 my-4">
+            <h2 class="text-lg font-bold">Create user</h2>
+            <label class="label">
+                Full name
+                <input
+                    type="text"
+                    name="full_name"
+                    v-model="newUser.full_name"
+                    autocomplete="name"
+                    class="input"
+                />
+            </label>
+            <label class="label">
+                Password
+                <input
+                    type="password"
+                    name="password_hash"
+                    v-model="newUser.password_hash"
+                    autocomplete="name"
+                    class="input"
+                />
+            </label>
+            <label class="label">
+                Email
+                <input
+                    type="email"
+                    name="email"
+                    v-model="newUser.email"
+                    autocomplete="name"
+                    class="input"
+                />
+            </label>
+
+            <label
+                ><input
+                    type="submit"
+                    class="button"
+                    :disabled="loading"
+                    value="Create user"
+            /></label>
+        </form>
     </div>
 </template>
 
 <script>
 export default {
+    methods: {
+        async submit() {
+            await fetch('/api/user', {
+                headers: {
+                    'Content-Type': 'application/json',
+                    Accept: 'application/json',
+                },
+                method: 'POST',
+                body: JSON.stringify(this.newUser),
+            })
```

Comment:
Maybe make an API-client instead of this?
This especially is more usefull whenever we implement authentication.